### PR TITLE
Site administrators can POST

### PR DESCRIPTION
**Pull request by daisieh**
Passes pytest/travis. If you've got site_admin credentials set up on a user (pull `develop` after https://github.com/CanDIG/CanDIGv2/pull/139 and run `make clean-authx` and `make init-authx` again), user2 should be able to create datasets/projects/tables in katsu, but user1 shouldn't be able to.

### CODE REVIEW

File: chord_metadata_service/restapi/candig_authz_middleware.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                    error_response = {
+                        "error": "This request failed because caching is not disabled. Please contact your system administrator for assistance."
+                    }
+                    response = HttpResponseForbidden(json.dumps(error_response))
+                    response["Content-Type"] = "application/json"
+                    return response
+                opa_res_datasets = self.get_opa_datasets(token, request.path, request.method)
+                if len(opa_res_datasets) == 0:
+                    self.authorize_datasets = 'NO_DATASETS_AUTHORIZED'
+                elif type(opa_res_datasets) == tuple and opa_res_datasets[0] == "error":  # error response
```
Comment (shaikh-rashid) [inline chord_metadata_service/restapi/candig_authz_middleware.py:42]: Concise error code

File: chord_metadata_service/restapi/candig_authz_middleware.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                    return opa_res_datasets[1]
```
Comment (shaikh-rashid) [inline chord_metadata_service/restapi/candig_authz_middleware.py:43]: So OPA sends an error and the next element is the error message or a status code?

**Review by shaikh-rashid — APPROVED** [review]
Looks good, had one question regarding OPA datasets response handling but it is not a blocking or request for change.

File: chord_metadata_service/restapi/candig_authz_middleware.py (hunk 2 above)
Comment (daisieh) [reply chord_metadata_service/restapi/candig_authz_middleware.py:43]: Yeah, an error message

**Pull request merged**